.github/workflows/issues-workflow.yml

```
@@ -0,0 +1,208 @@
+          project_id: barterverse-l9uq3
+      
+      - name: Auto-label new issues
+        if: github.event.action == 'opened'
+        run: |
+          GITHUB_TOKEN=$(gcloud secrets versions access latest --secret="github-token" --project="barterverse-l9uq3")
+          
+          # Get issue details
+          ISSUE_BODY="${{ github.event.issue.body }}"
+          ISSUE_TITLE="${{ github.event.issue.title }}"
+          ISSUE_NUMBER="${{ github.event.issue.number }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)